src/nova_cors_plugin.erl

```diff
@@ -2,24 +2,23 @@
 -behaviour(nova_plugin).
 
 
--export([pre_http_request/2,
-         post_http_request/2,
+-export([pre_request/2,
+         post_request/2,
          plugin_info/0]).
 
-pre_http_request(#{req := Req} = NovaState, _) ->
+pre_request(Req, _) ->
     Req1 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Origin">>, <<"*">>, Req),
     Req2 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Headers">>, <<"*">>, Req1),
     Req3 = cowboy_req:set_resp_header(<<"Access-Control-Allow-Methods">>, <<"*">>, Req2),
-    NewState = maps:put(req, Req3, NovaState),
-    case Req of
+    case Req3 of
```

Comment:
Wouldn't it make more sense to pattern match against the <<"OPTIONS">> method instead?

Reply:
Doesn't we need the cors even on OPTIONS? And patter match where on what?

Reply:
I'm not 100% sure tbh if we need it for OPTIONS. But now I just realized that my idea about pattern patching was wrong. Sorry about that!